lib/util/twilio-video-errors.js

```diff
@@ -242,6 +344,142 @@ exports.RoomNotFoundError = RoomNotFoundError;
+ * @extends TwilioError
+ * @property {number} code - 53110
+ * @property {string} message - 'StatusCallbackMethod is invalid'
+ */
+function RoomStatusCallbackMethodInvalidError() {
+  TwilioError.call(this,
+    53110,
+    'StatusCallbackMethod is invalid'
+  );
+}
+inherits(RoomStatusCallbackMethodInvalidError, TwilioError);
+exports.RoomStatusCallbackMethodInvalidError = RoomStatusCallbackMethodInvalidError;
+Object.defineProperty(TwilioErrorByCode, 53110, { value: RoomStatusCallbackMethodInvalidError });
+
+/**
+ * @class RoomStatusCallbackInvalidError
+ * @classdesc Raised in the REST API when StatusCallback is not a valid URL or the url is too long.
+ * @extends TwilioError
+ * @property {number} code - 53111
+ * @property {string} message - 'StatusCallback is invalid'
+ */
+function RoomStatusCallbackInvalidError() {
+  TwilioError.call(this,
+    53111,
+    'StatusCallback is invalid'
+  );
+}
+inherits(RoomStatusCallbackInvalidError, TwilioError);
+exports.RoomStatusCallbackInvalidError = RoomStatusCallbackInvalidError;
+Object.defineProperty(TwilioErrorByCode, 53111, { value: RoomStatusCallbackInvalidError });
+
+/**
+ * @class RoomStatusInvalidError
+ * @classdesc Raised in the REST API when Status is not valid or the Room is not in-progress.
+ * @extends TwilioError
+ * @property {number} code - 53112
+ * @property {string} message - 'Status is invalid'
+ */
+function RoomStatusInvalidError() {
+  TwilioError.call(this,
+    53112,
+    'Status is invalid'
+  );
+}
+inherits(RoomStatusInvalidError, TwilioError);
+exports.RoomStatusInvalidError = RoomStatusInvalidError;
+Object.defineProperty(TwilioErrorByCode, 53112, { value: RoomStatusInvalidError });
+
+/**
+ * @class RoomRoomExistsError
+ * @classdesc Raised in the REST API when the Room creation fails because a Room exists with the same name.
+ * @extends TwilioError
+ * @property {number} code - 53113
+ * @property {string} message - 'Room exists'
+ */
+function RoomRoomExistsError() {
+  TwilioError.call(this,
+    53113,
+    'Room exists'
+  );
+}
+inherits(RoomRoomExistsError, TwilioError);
+exports.RoomRoomExistsError = RoomRoomExistsError;
+Object.defineProperty(TwilioErrorByCode, 53113, { value: RoomRoomExistsError });
+
+/**
+ * @class RoomInvalidParametersError
+ * @classdesc Raised in the REST API when one or more Room creation parameter is incompatible with the Room type.
+ * @extends TwilioError
+ * @property {number} code - 53114
+ * @property {string} message - 'Room creation parameter(s) incompatible with the Room type'
+ */
+function RoomInvalidParametersError() {
+  TwilioError.call(this,
+    53114,
+    'Room creation parameter(s) incompatible with the Room type'
+  );
+}
+inherits(RoomInvalidParametersError, TwilioError);
+exports.RoomInvalidParametersError = RoomInvalidParametersError;
+Object.defineProperty(TwilioErrorByCode, 53114, { value: RoomInvalidParametersError });
+
+/**
```

Comment:
Hmm, these should never be raised at the client if I am not mistaken. Maybe we can skip these? We may need a way to identify which errors can and should not be raised at the Client...

Reply:
@markandrus On second thought, I think we should hold off on removing certain errors manually until we figure out a way to generate only client specific errors. I'm sure there are other errors apart from this that we might have to remove.